runtime/src/main/scala/akka/grpc/javadsl/GrpcExceptionHandler.scala

```diff
@@ -7,12 +7,16 @@ package akka.grpc.javadsl
 import java.util.concurrent.CompletionException
 
 import akka.grpc.GrpcServiceException
-
-import scala.concurrent.ExecutionException
 import akka.http.javadsl.model.HttpResponse
 import io.grpc.Status
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.ExecutionException
 
 object GrpcExceptionHandler {
+
+  private val log = LoggerFactory.getLogger(getClass)
```

Comment:
Rather than creating an slf4j logger statically, would it be possible to pass in an `akka.event.LoggingAdapter`? That might add a little more complexity, but also make things more flexible.

Reply:
I prefer less complexity to more flexibility. It would not be better if we needed to do this?

Reply:
This is indeed a tricky balance. Certainly as a first step it is usually smart to prefer simplicity over flexibility.

On the other hand, simplicity comes in many forms: isn't is simpler if akka-grpc-runtime depends only on Akka API's, and not also on the slf4j API's? In this case this could be especially relevant, since indeed akka-grpc-runtime doesn't introduce a dependency on slf4j, so people running without slf4j would get errors. The only reason this compiles is because we haven't had the time yet to move play-specific code to a separate project, so play (and thus slf4j) is included as 'provided' in akka-grpc-runtime.

Reply:
Of course you could argue slf4j is so common it should always be included. This was not so clear when Akka started, but indeed we are actively considering making the slf4j dependency mandatory in Akka and having it replace some of Akka's own API's (https://github.com/akka/akka-meta/issues/31). We have not yet determined what would be the best way to achieve that, however, and there are a couple of features of Akka's logging subsystem for which it is not completely obvious how they could be replaced with slf4j.

Reply:
Until that time I think it would be better to avoid slf4j and use the Akka API's in akka-grpc-runtime, however.

Reply:
Ok I'll fix it